### PR TITLE
[Admin] Add base form type for every resource used in AdminBundle

### PR DESCRIPTION
**Pull request by TheMilek**
| Q               | A
|-----------------|-----
| Branch?         | 1.14 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| License         | MIT

related to https://github.com/Sylius/Sylius/pull/16257

### CODE REVIEW

**Review by Wojdylak — DISMISSED** [review]

**Review by mpysiak — DISMISSED** [review]

File: src/Sylius/Bundle/AdminBundle/Form/Type/CatalogPromotionType.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('scopes', CollectionType::class, [
+        'label' => 'sylius.ui.scopes',
+        'entry_type' => CatalogPromotionScopeType::class,
+        'allow_add' => true,
+        'allow_delete' => true,
+        'by_reference' => false,
+        'required' => false,
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Form/Type/CatalogPromotionType.php]: Missing indentation

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/form.xml [hunk 2]
```diff
@@ -36,6 +36,122 @@
...
+            <tag name="form.type" />
+        </service>
+
+        <service id="sylius_admin.form.type.customer_group" class="Sylius\Bundle\AdminBundle\Form\Type\CustomerGroupType" >
+            <tag name="form.type" />
+        </service>
+
+        <service id="sylius_admin.form.type.customer" class="Sylius\Bundle\AdminBundle\Form\Type\CustomerType" >
+            <tag name="form.type" />
+        </service>
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/form.xml]: Add all services definitions in alphabetical order

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/form.xml [hunk 3]
```diff
@@ -36,6 +36,122 @@
...
+
+        <service id="sylius_admin.form.type.product" class="Sylius\Bundle\AdminBundle\Form\Type\ProductType" >
+            <tag name="form.type" />
+        </service>
+
+        <service id="sylius_admin.form.type.product_variant" class="Sylius\Bundle\AdminBundle\Form\Type\ProductVariantType" >
+            <tag name="form.type" />
+        </service>
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/form.xml]: As above

**Review by GSadee — COMMENTED** [review]
Since we are based on form types for resources, I would also think about:

- AvatarImageType
- ChannelPriceHistoryConfigType
- ChannelPricingType
- GatewayConfigType
- ProductAssociationType
- ProductAttributeTranslationType
- ProductImageType
- ProductOptionTranslationType
- ProductTranslationType
- ProductVariantTranslationType
- ProvinceType
- ShopBillingDataType
- TaxonImageType
- TaxonTranslationType 
- ZoneMemberType

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Kamil! :1st_place_medal:

**Pull request merged**